tests/src/test/java/org/sonarqube/tests/issue/AutoAssignTest.java

```diff
@@ -151,7 +151,12 @@ public void update_author_and_assignee_when_scm_is_activated() {
 
     // Authors and assignees are set
     for (Issue issue : issues) {
-      assertThat(issue.author()).isNotEmpty();
+      if (issue.line() == 10) {
+        // SONAR-8727
+        assertThat(issue.author()).isEmpty();
+      } else {
+        assertThat(issue.author()).isNotEmpty();
+      }
```

Comment:
I would prefer to make the test even more explicit, by using:
```
    assertThat(issues)
      .extracting(Issue::line, Issue::author)
      .containsExactlyInAnyOrder(
        tuple(1, "user1"),
        tuple(2, "user2"),
        tuple(3, "user3name"),
        tuple(4, "user4name"),
        tuple(5, "user5@email.com"),
        tuple(6, "user6@email.com"),
        tuple(7, "user7scmaccount"),
        tuple(8, "user8scmaccount"),
        tuple(9, "user8scmaccount"),
        tuple(10, "")
      );

```

(and on line 142:

```
    assertThat(issues)
      .extracting(Issue::line, Issue::author)
      .containsExactlyInAnyOrder(
        tuple(1, ""),
        tuple(2, ""),
        tuple(3, ""),
        tuple(4, ""),
        tuple(5, ""),
        tuple(6, ""),
        tuple(7, ""),
        tuple(8, ""),
        tuple(9, ""),
        tuple(10, "")
      );

```
)